modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/consensus/block/AlgebraCommand.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.dag.l1.domain.consensus.block
+
+import org.tessellation.dag.l1.domain.consensus.block.BlockConsensusData.{
+  BlockProposal,
+  CancelledBlockCreationRound,
+  Proposal
+}
+import org.tessellation.kernel.Ω
+
+sealed trait AlgebraCommand extends Ω
```

Comment:
@marcinwadon `RoundStarFailure` is not a step as well  :D

Reply:
right @kpudlik, it's either an outcome/result or an error

Reply:
Right I can change the name to `InformAboutRoundStartFailure` - it would be consistent then.
Regarding the naming I think we should use the same convention across the project, though I must admit I like the `Command` convention a lot. `Step` is kind of passive/impersonal for me, whereas `Command` is more vibrant and suiting to what's actually happening - that's how I use it also `PersistBlockProposal`, `StartOwnRound` etc. For me that's more descriptive. Also a step suggests that after you make first step, you will then make a second one, whereas here it's not true - you can make the same step many times (process proposal for one peer than for the other etc.) and also the cell is stateless in a way (it reads and modifies the data living in a storage) so in a way it can even go backwards (it's not tied to the concrete state). That's my thoughts. WDYT?